translation/source/mobile.xml

```diff
@@ -6,6 +6,7 @@
   <string name="areYouSure" comment="This is shown in confirmation dialogs.">Are you sure?</string>
   <string name="boardSettings" comment="This is shown in the app main settings screen, which opens a screen dedicated to chess board settings.">Board settings</string>
   <string name="cancelTakebackOffer" comment="Menu action that cancels a previous takeback offer">Cancel takeback offer</string>
+  <string name="casualGame" comment="This is shown in quick actions to show the name is casual">(Casual)</string>
```

Comment:
There is already a `casual` translation key that we can use. So please remove this one, we want to avoid adding extra work to translators.